renovate.json

```diff
@@ -17,6 +17,6 @@
   ],
   "rangeStrategy": "pin",
   "constraints": {
-    "node": ">=16.0.0 <17.0.0"
+    "node": ">=18.0.0 <19.0.0"
```

Comment:
can you gwt rid of the max here, it really doesnt matter here and 20 is lts

Reply:
Updated in e8b5363